### ISSUE TITLE
Missing LICENSE file

### ISSUE THREAD

**Issue by srakitnican** [AGProjects/python-eventlib#1]
Under what license is this project under?

Please add LICENSE file.

**Comment by danpascu** [comment]
LGPL. The license is specified in debian/copyright.

I would not recommend anyone use this project. It's an old fork of python-eventlet which we did in order to preserve the twisted hub which was removed from eventlet, but we needed it in python-sipsimple. It is our intention to replace this and get rid of it when we'll find the time.

**Comment by srakitnican** [comment]
Thanks for the info. I am packaging it precisely because it's needed by `python-sipsimple`, going back to `python-eventlet` would be great!

Since it is only required by `python-sipsimple` do you think it makes sense to bundle it along with `python-sipsimple` when packaging?

Can you tell me about other python libraries provided by AGProjects and required by `python-sipsimple`/`blink-qt`, are those forks of some sort as well or original sources?
```
python-msrplib
python-otr
python-xcaplib
```

**Comment by danpascu** [comment]
It's not only required by python-sipsimple. It is used by many of our projects, including python-msrplib, python-xcaplib, python-sipsimple, blink, openxcap and possible others, so no it cannot be bundled.

Except python-eventlib which is a fork of eventlet, everything else is original code.

**Comment by srakitnican** [comment]
Am I understanding right, you are saying that `python-eventlib` is currently used by other libraries and blink as well, not just `python-sipsimple`?

**Comment by danpascu** [comment]
Yes, at least the ones I listed. I don't remember all of them of the top of my head.
